packages/@coorpacademy-components/src/molecule/share/index.js

```diff
@@ -93,13 +93,15 @@ const Share = ({text, wording}) => {
   }, [setStatus]);
 
   return (
-    <div className={styles.container}>
-      <Link onClick={onClick} className={styles.cta}>
-        <div className={styles.wrapper}>
-          <ShareIcon className={styles.shareIcon} width={18} height={18} />
-          <p>{wording}</p>
-        </div>
-      </Link>
+    <div className={style}>
+      <div className={styles.container}>
```

Comment:
pour quoi ajouter une div de plus
```suggestion
      <div className={classNames(style, styles.container)}>
```